README.md

```diff
@@ -36,39 +36,21 @@ msgParams should have a `data` key that is hex-encoded data unsigned, and a `sig
 
 Returns a hex-encoded sender address.
 
-### signTypedDataLegacy (privateKeyBuffer, msgParams)
+### signTypedData (privateKeyBuffer, msgParams, version)
 
-Signs typed data as per [an early draft of EIP 712](https://github.com/ethereum/EIPs/pull/712/commits/21abe254fe0452d8583d5b132b1d7be87c0439ca).
+Sign typed data according to EIP-712. The signing differs based upon the `version`, which is explained in the table below.
 
 Data should be under `data` key of `msgParams`. The method returns prefixed signature.
 
-### signTypedData (privateKeyBuffer, msgParams)
+| Version | Description                                                  |
+| ------- | ------------------------------------------------------------ |
+| `V1`    | This is based on [an early version of EIP-712](https://github.com/ethereum/EIPs/pull/712/commits/21abe254fe0452d8583d5b132b1d7be87c0439ca) that lacked some later security improvements, and should generally be neglected in favor of `V3`. |
```

Comment:
should we add a note about the lack of `V2`?

Reply:
Probably, yep! I just didn't know what to say - I have no idea why there isn't a V2

Reply:
Oh, actually, I just found more information on this in our docs:

>The missing `v2` represents an intermediary design that was implemented by the Cipher browser, so that we have room to implement it if there is ever enough developer demand for it.

I'll add this in a follow-up PR.